bionic/flow.py

```diff
@@ -844,46 +907,60 @@ def to_builder(self):
 
         return FlowBuilder._from_state(self._state)
 
-    def get(self, name, fmt=None):
+    def get(self, name, collection=None, object_type=object):
```

Comment:
I like the name change of `collection`, but it might be nice to keep the original `fmt` parameter but deprecate it (i.e., keep it working but emit a warning) -- just in case anyone has code that uses the keyword explicitly.

Reply:
Ah, so just to clarify: are you suggesting we temporarily allow both but emit a deprecation warning if `fmt` is used?  This ensures code relying on `fmt` will still work for now and we can remove in the future, fully moving to `collection`?

Reply:
Yep, exactly!